ietf/doc/utils.py

```diff
@@ -771,7 +771,11 @@ def rebuild_reference_relations(doc, filenames):
     unfound = set()
     for ( ref, refType ) in refs.items():
         # As of Dec 2021, DocAlias has a unique constraint on the name field, so count > 1 should not occur
-        refdoc = DocAlias.objects.filter( name=ref )
+        refdoc = (
+            DocAlias.objects.filter(name=ref[:-3])
+            if re.match(r"^draft-.*-\d{2}$", ref)
```

Comment:
This falls into trouble if the draft being referenced has a name that actually ends with -nn, such as `draft-ietf-tsvwg-ieee-802-11`. The pattern we've had before is to look to see if there's a matching name with the -nn first and then check to see if there's a match without it only if the first doesn't exist.